apps/docs/content/guides/telemetry/metrics.mdx

```diff
@@ -28,13 +28,39 @@ Your project's metrics endpoint is accessible at `https://<project-ref>.supabase
 > curl https://<project-ref>.supabase.co/customer/v1/privileged/metrics --user 'service_role:<service-role-jwt>'
 ```
 
-## Configuring Prometheus
+## Supabase Grafana
 
-For self-hosted Prometheus, modify your `prometheus.yaml` file to add a Supabase job, and set the `metrics_path`, `scheme`, `basic_auth` and `targets` parameters. For example:
+The pre-configured Supabase Grafana Dashboard is an advanced version of the [Dashboard's Database Reports](https://supabase.com/dashboard/project/_/reports/database). It visualizes over 200 database performance and health metrics and is updated every minute.
```

Comment:
> ...updated every minute.

I don't know how to phrase it better, but one might think new metrics will be coming every minute, but I think that depends on prometheus scraping interval and what is meant here is that dashboard is refreshed every minute (also could be changed by a user).
Or did I get it wrong?

Reply:
You are right, those things should be separated!
- It does mean the metrics endpoint is refreshed every minute (so the scrape job should be 60s)
- The dashboard probably should not mention the refresh time as it is not configured to auto refresh by default